workbook/04-contributor.asciidoc

```diff
@@ -0,0 +1,178 @@
+== Video:  InnerSource Contributor
+
+=== SEGMENT: Becoming an InnerSource Contributor
+
+TIP:
+More than one answer may be correct in some questions.
+
+==== Question 1.  What is a benefit of becoming a contributor to an InnerSource project?
+
+. I can make sure others have to follow my suggestions to improve the project.
+. The host team will maintain the features I add on behalf of my team.
+. I can shape the solution to better fit my team’s needs.
+. I can help to shape all aspects of the project beyond the code itself (for example, GitHub branches and merges, tests, documentation).
+
+
+===== Correct answers 3 and 4
+
+Why 1 is incorrect: Open communities are not a dictatorship. As a contributor, you are part of a community helping the solution to grow and thrive.  This may involve compromise at times, but will pay longer term benefits to you and the community. 
+
+Why 2 is incorrect: In a shared solution you are invited to help improve the code, which includes bug fixes. Ownership in shared solutions should be seen as stewardship to keep the solution long term healthy.
+
+Why 3 is correct: Shared solutions develop out of contributions from interested parties. Becoming a contributor allows you to shape the solution, always in consultation and collaboration with other community members.
+
+Why 4 is correct: Contributions go beyond just code. There are many aspects that help to keep a solution healthy and thus make it successful, different aspects require different skills, though don’t make one more important than another. If the code is the machine, then think of these areas of contributions as the grease that keeps the machine running smoothly.
+
+==== Question 3.  What is a productive assumption for a contributor to make when joining an InnerSource project?
+
+. I need to make sure to communicate my intent when adding or changing the project.
+. I need to make sure my code serves the hosting teams goals.
+. If the hosting team is in a bind to meet their deadlines, I have to spend time to code for them.
+. I should work only on features needed by my team.
+
+===== Correct answer: 1
+
+Why 1 is correct: If your contribution isn’t understood by the host team or the community at large, so they may reverse your hard work or make other changes that weaken the value of your contribution. Open and early communication with the community and the hosting team will help you to drive successful contributions.
+
+Why 2 is incorrect: You should make sure that your goals do not conflict with those of the hosting team, but beyond that you can still promote your own team’s goals.
+
+Why 3 is incorrect: Your work on another team’s project reflects your own team’s needs, but you are not working for the other team. If the hosting team (or any community participant) is in a bind, you and your management might offer help as a gesture to benefit the whole company., However, it is not your task to make up for the time or staffing shortcomings at the hosting team.
```

Comment:
I like this explanation that balances the option to  help against having to help!